ovos_plugin_manager/utils/audio.py

```
@@ -0,0 +1,531 @@
+                    "--stdout", "--totally-silent",
+                    # put the resulting AIFF file in stdout, and make sure it's not mixed with any program output
+                    "--decode", "--force-aiff-format",  # decode the FLAC file into an AIFF file
+                    "-",  # the input FLAC file contents will be given in stdin
+                ], stdin=subprocess.PIPE, stdout=subprocess.PIPE, startupinfo=startup_info)
+                aiff_data, _ = process.communicate(flac_data)
+                aiff_file = io.BytesIO(aiff_data)
+                try:
+                    self.audio_reader = aifc.open(aiff_file, "rb")
+                except (aifc.Error, EOFError):
+                    raise ValueError(
+                        "Audio file could not be read as PCM WAV, AIFF/AIFF-C, or Native FLAC; check if file is corrupted or in another format")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)